assets/javascripts/discourse/widgets/discourse-post-event-dates.js

```diff
@@ -15,27 +15,28 @@ export default createWidget("discourse-post-event-dates", {
     });
   },
 
-  html(attrs) {
+  html({ localDates, eventModel }) {
     const content = [
       iconNode("clock"),
-      h("span.date", new RawHtml({ html: `<span>${attrs.localDates}</span>` })),
+      h("span.date", new RawHtml({ html: `<span>${localDates}</span>` })),
     ];
 
     if (
-      attrs.eventModel.is_expired &&
-      attrs.eventModel.status !== "standalone"
+      eventModel.is_expired &&
```

Comment:
Ensures we only show participants when the event was not "standalone" nor closed.